source/jormungandr/jormungandr/scenarios/ridesharing/ouestgo.py

```diff
@@ -211,6 +188,12 @@ def _request_journeys(self, from_coord, to_coord, period_extremity, instance_par
             'p[to][longitude]': arr_lon,
             'signature': 'toto',
             'timestamp': period_extremity.datetime,
+            'p[outward][mindate]': timestamp_to_date_str(
+                period_extremity.datetime, timezone, _format="%Y-%m-%d"
```

Comment:
Minor: looks better to use the already defined one (or let the default one?)
```suggestion
                period_extremity.datetime, timezone, _format=DATE_FORMAT
```

Reply:
:+1: